main.go

```diff
@@ -129,8 +138,11 @@ func main() {
 	flag.BoolVar(&configAuditEnabled, "config-audits-enabled", true,
 		"Enable metrics for ConfigAuditReport resources.")
 
-	flag.BoolVar(&vulnerabilityScansEnabled, "vulnerability-scans-enabled", true,
-		"Enable metrics for VulnerabilityReport resources.")
+	flag.BoolVar(&trivyVulnerabilityScansEnabled, "trivy-vulnerability-scans-enabled", true,
+		"Enable metrics for Trivy VulnerabilityReport resources.")
+
+	flag.BoolVar(&kubescapeVulnerabilityScansEnabled, "kubescape-vulnerability-scans-enabled", true,
+		"Enable metrics for KubescapeVulnerabilityReport resources.")
```

Comment:
Team WDYT? This is a breaking change. Do we change flags, or keep one and set the scanner config in a config file?

Reply:
We chatted. IMO best case is there's a single generic flag, plus a path to a config file. But I'm not mean enough to make @peterspr implement that to merge this.

I agree Trivy remains the default until we switch -- this implementation keeps Trivy AND kubescape on by default, but the flag weirdness is hidden by the chart.

So, I'm fine breaking the flags, because for our customer usage they'll be hidden by Helm anyway, and we discussed ensuring that existing values will keep working (though this is likely going out as a major after this anyway).